cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ConnectorStatusUpdater.java

```diff
@@ -108,5 +112,26 @@ private void measure(ManagedConnector connector, Integer connectorState) {
             .tags(tags)
             .register(registry);
 
+        Counter.builder(config.metrics().baseName() + "." + CONNECTOR_STATE_COUNT)
+            .tags(tags)
+            .tag("cos.connector.state", ConnectorStatusExtractor.extract(connector).getPhase().getValue())
+            .register(registry)
+            .increment();
+
+        if (CONNECTOR_STATE_FAILED.compareTo(connectorState) == 0) {
```

Comment:
this can probably be replace by equals but maybe all the constants can be switched to a primitive type (int instead of Integer)